howso/analysis.amlg

```diff
@@ -508,9 +530,158 @@
 			)
 		)
 
+		(if (and
+				use_submodel
+				;need the submodel if using deviations
+				(get baseline_hyperparameter_map "featureDeviations")
+				(not !dynamicDeviationsDuringAnalyze)
+			)
+			; store feature residuals for a collection of cases at end, if it wasn't part of iterative process
+			(call !StoreResidualSubmodel (assoc custom_hyperparam_map baseline_hyperparameter_map) )
+		)
+
 		(call !SetModelHyperParameters)
 	)
 
+	;helper method for Analyze that stores a submodel of case residuals for each feature
+	;parameters:
+	;	custom_hyperparam_map: the hyperparam map within analyze that should be used to run the residuals
+	;assigns to:
+	;	deviation_submodel_name: this variable will store the name of the submodel, this name should be stored in the
+	;					resulting hyperparameter assoc under the key "submodelName"
+	#!StoreResidualSubmodel
```

Comment:
Should we use some form of the term Subtrainee instead of Submodel, to match our external direction to not use the word model?

Reply:
subtrainee now